README.md

```diff
@@ -48,265 +48,152 @@ The FSXA-API is subject to the Apache-2.0 license.
-```
-
-### buildAuthorizationHeaders
-
-Returns the build authorization header in the following format when mode is set to `remote`:
+Returns the build authorization header in the following format when using `FSXARemoteApi`:
 
 ```typescript
 {
   authorization: 'apikey="xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"'
 }
 ```
 
-Returns an empty object when mode is set to `proxy`.
-
 Example:
 
 ```typescript
-fsxaApi.buildAuthorizationHeaders()
+const remoteApi = new FSXARemoteApi(config)
+const authorizationHeader = remoteApi.authorizationHeader
+// console.log { authorization: `apikey="${config.apikey}"` }
 ```
 
-### buildCaaSURL
-
-Expects an optional parameter `remoteProjectId`. If passed, the CaaS Url will be built with it instead of the default project id.
-
-Returns the build CaaS url when mode is set to `remote`:
+### fetchNavigation
 
-Returns an empty string when mode is set to `proxy`.
+This method fetches the navigation from the configured navigation service. You need to pass a `FetchNavigationParams` object.
 
 Example:
 
 ```typescript
-fsxaApi.buildCaaSUrl()
-```
-
-### buildNavigationServiceUrl
-
-Returns the build navigation-service url when mode is set to `remote`:
-
-Returns an empty string when mode is set to `proxy`.
-
-Example:
-
-```typescript
-fsxaApi.buildNavigationServiceUrl()
+fsxaApi.fetchNavigation({
+  locale: 'en_EN',
+  initialPath: '/',
+})
 ```
 
 ### fetchElement
 
-Returns the corresponding CaaS data entry.
-
-Expects as input parameter an id, which is described in CaaS as 'identifier' and a language abbreviation.
-<br />
-Optionally, additional parameters can be passed that will be appended to the CaaS-Url. Be aware that the response is not mapped if you pass the keys. For more information please refer to the [restheart documentation](https://restheart.org/docs/read-docs/#projection).
-<br />
-Optionally, a remoteProject can be passed. If one is passed, the element will be fetched from this project instead of the default one.
-
-In this example the additional parameters ensure that only the fields `identifier` and `displayName` are in the result set:
+This method fetches an element from the configured CaaS. The `FetchElementParams` object defines options to specify your request. Check `FSXARemoteApi.buildCaaSUrl` to know which URL will be used.
 
 ```typescript
-fsxaApi.fetchElement('xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx', 'en-EN', {
-  keys: [{ identifier: 1 }, { displayName: 1 }],
+fsxaApi.fetchElement({
+  id: 'xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx',
+  locale: 'de_DE',
 })
 ```
 
+> **_Note_**<br>
+> The 'raw' CaaS data might be helpful when you want to know the key names that you can filter for.
+> It is possible to access those data directly with the authorization header and the CaaS Url.
+
 ### fetchByFilter
 
 Returns the matching CaaS data entries.
+It maps the given entries to a more frontend-friendly format.
+Which means that unneccessary fields will be omitted and the structure is simpler.
```

Comment:
Unnecessary ...

Reply:
for the frontend developers unneccessary. Which word, would you like to use here?